README.md

```diff
@@ -1,6 +1,6 @@
 # Slice
 [![Maven Central](https://img.shields.io/maven-central/v/io.airlift/slice.svg?label=Maven%20Central)](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22io.airlift%22%20AND%20a%3A%22slice%22)
-[![Build Status](https://travis-ci.org/airlift/slice.svg?branch=master)](https://travis-ci.org/airlift/slice)
+[![Build Status](https://github.com/airlift/slice/workflows/CI/badge.svg)](https://github.com/airlift/slice/actions?query=workflow%3ACI+event%3Apush+branch%3Amaster)
```

Comment:
This is fine, but I would simply remove this. People can see the green checkmark for the latest commit if they care.